gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,7 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
In this doc, it suggests to use `8.2.1` as the Gradle version.
https://docs.google.com/document/d/1Pdw-FJ9l1Kzd3HlNFz3pPTcdKTgcvbpRkoV6aliwkcU/edit#heading=h.5k2ixztyrgo8

Reply:
We don't have to restrict to this version IMO. This should be taken as a minimal viable Gradle version rather than the strict version we should use.